### ISSUE TITLE
Characters too Shiny 

### ISSUE THREAD

**Issue by Kitakazi** [TheGameCreators/GameGuruRepo#1675]
I believe this was reported before and was fixed? Not sure but characters are mega shiny, looks like they have an outline or something to them.  Even indoors it is very noticeable
<img width="1280" alt="GameGuruMAX Screenshot 2021 12 16 - 19 04 09 92" src="https://user-images.githubusercontent.com/51464873/146472697-6ea894eb-1359-4bad-95de-c31ccbf28284.png">
<img width="1280" alt="GameGuruMAX Screenshot 2021 12 16 - 19 02 51 82" src="https://user-images.githubusercontent.com/51464873/146472702-be305e41-95de-4fa3-873e-d35fef4c02f4.png">

**Comment by fearlesswee** [comment]
This appears to be a "rimlighting" effect that can be somewhat resolved by placing a light, reducing it's light range to 0, and probe range to max to make an "environment probe" which corrects the lighting. It's still noticeable a bit however, and placing TONS of probes over the entire 5km x 5km area would probably not really be a "fix" lol!

I was the one that reported this a while ago, and it hadn't been fixed yet. (Well, you can probably see that lol)

**Comment by cstriage13** [comment]
Subsurface scattering would fix this...

**Comment by Kitakazi** [comment]
While subsurface scattering might help make the characters look better I think it is caused by some sort of effect that is on the character models. Like Fearlesswee commented it could be some sort of rimlight effect or a different shader is being used for characters. But w/e it is it looks very cartoony and strange.  In my screenshots look at the pants, there is a right of light around them. It shouldnt look like that.

**Comment by cstriage13** [comment]
Yes it is the same effect i am getting from the trees on the pants. I feel
like it is the fog.

**Comment by cstriage13** [comment]
Look at my bug I found with the trees like they are not compositing
correctly its the same thing but at the distance it goes away largely. I
was playing with the fog last night and those are the settings that make it
seam worse.

**Comment by cstriage13** [comment]
Another thing I would like to add is that subsurf would help the face not
be so sharp/cartoony. It would lighten the lines and work with antialiasing
to soffen the edges. Also what is your system is it running off the board
or your gfx card. I know i had to set it up to use my gfx for ggm.

**Comment by Kitakazi** [comment]
Characters look better now. I see no more glow-like outline in latest build.
